tests/beta/queuing/test_beta_measure.py

```diff
@@ -48,8 +49,11 @@ def test_annotating_obs_return_type(self, stat_func, return_type, op):
             A = op(0)
```

Comment:
Just noticed, but the docstrings in these tests can be updated, since they cover more than just "expval" calls

Reply:
Thanks, updated it!